packages/web-client/app/components/card-pay/deposit-workflow/transaction-status/index.ts

```diff
@@ -36,6 +45,37 @@ class CardPayDepositWorkflowTransactionStatusComponent extends Component<Workflo
 
   constructor(owner: unknown, args: WorkflowCardComponentArgs) {
     super(owner, args);
+    taskFor(this.waitForBlockConfirmationsTask)
+      .perform()
+      .catch((e) => {
+        this.blockCountError = true;
+        console.error(e);
+      });
+  }
+
+  get displaySubstate() {
+    return taskFor(this.waitForBlockConfirmationsTask).isRunning;
+  }
+
+  get substate() {
```

Comment:
`substate` is pretty generic. This might be clearer if it was named `progressSubstate` or `bridgingSubstate`. Feel free to ignore if you disagree.